.eslintrc.js

```diff
@@ -39,8 +39,10 @@ module.exports = {
     ],
     '@typescript-eslint/no-redeclare': 'off',
 
-    // We LOVE non-breaking spaces here
+    // We LOVE non-breaking spaces, and both straight and curly quotes here
     'no-irregular-whitespace': [1, { skipJSXText: true, skipStrings: true }],
+    'react/no-unescaped-entities': [1, { forbid: ['>', '}'] }],
+    'material-ui/straight-quotes': 'off',
```

Comment:
We need to be able to write text with proper typography in JSX

Reply:
FYI, there's a discussion on Notion about this: https://www.notion.so/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b?d=aff8a8dc579c48f8a2500de4253c6949&pvs=4#b17ab6d6eb6f4589ba084207caf33443

Reply:
2022! 😀

Reply:
Does anything changed significantly in this matter since then?